public/src/pages/views/MemberView.tsx

```diff
@@ -45,18 +45,17 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
                     onChange={(query) => this.handleSearch(query)}
                 />
             {this.renderPendingView(pendingActions)}
-            {this.props.pendingUsers.length > 0 ? this.approveButton() : null}
             {this.renderUserView()}
             {this.renderRejectedView()}
         </div>;
     }
 
-    public componentDidUpdate(prevProps: IUserViewerProps) {
-        if ((prevProps.acceptedUsers.length !== this.props.acceptedUsers.length)
-         || (prevProps.pendingUsers.length !== this.props.pendingUsers.length)
-          || (prevProps.rejectedUsers.length !== this.props.rejectedUsers.length)) {
-            return this.refreshState();
-       }
+    public componentWillReceiveProps(newProps: IUserViewerProps) {
+        this.setState({
+            acceptedUsers: newProps.acceptedUsers,
+            rejectedUsers: newProps.rejectedUsers,
+            pendingUsers: newProps.pendingUsers,
+        });
     }
 
     public renderRejectedView() {
```

Comment:
Is this still used? (now that we reject users from a course... and delete the enrollment from the database.)